jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/GenericErrorSummaryCountCell.java

```diff
@@ -82,8 +90,13 @@ public void render(Context context,
         sb.appendHtmlConstant(tooltip.getOuterHTML());
     }
 
-    public void openErrorView(final String pid) {
-        viewPresenter.openErrorView(pid);
+    public void openErrorView(final String pid){
+        final PlaceRequest request = new DefaultPlaceRequest(EXECUTION_ERRORS);
+        request.addParameter(SEARCH_PARAMETER_PROCESS_INSTANCE_ID,
+                             pid);
+        request.addParameter(SEARCH_PARAMETER_IS_ERROR_ACK,
+                             Boolean.toString(false));
```

Comment:
@mcivantos-tribalyte @cristianonicolai I think we need to add another parameter to the active filters list in this case, so it matches the DB query we making in _DataSetDefsBootstrap_ and that is **Type: Task**.
Otherwise, navigation to Execution errors view using error count tooltip can show a different number of errors from the one errors column is showing in the task admin view, which would look inconsistent.

Reply:
+1

Reply:
Done.